sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/autorest.md

```diff
@@ -6,15 +6,16 @@ Run `dotnet build /t:GenerateCode` to generate code.
 > see https://aka.ms/autorest
 
 ``` yaml
-tag: package-artifacts-2019-06-01-preview
+tag: package-artifacts-2021-06-01-preview
```

Comment:
Why did you change the package tag as part of this autorest change?

Reply:
`package-artifacts-2019-06-01-preview` has an error that can't be mitigated by `transform` directive. So either we upgrade the tag or downgrade the version of used autorest package to the old one.

Reply:
Sounds good.  I have a change pending to upgrade to this tag, so I'll merge over this once it's committed, unless you'd like me to get that change in first.  Thanks!